keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -176,7 +177,7 @@ class TitusClusterHandler(
           "cluster" to moniker.name
         ),
         "reason" to "Diff detected at ${clock.instant().iso()}",
-        "type" to if (current == null) "createServerGroup" else "upsertServerGroup",
```

Comment:
apparently "upsertServerGroup" is not a thing... anyways changing to what `ClusterHandler` does